package.json

```diff
@@ -82,8 +82,19 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@babel/preset-react": "^7.14.5",
+    "@heroicons/react": "^1.0.4",
+    "algoliasearch": "^4.10.5",
+    "cnbuilder": "^2.7.1",
+    "decanter-react": "^1.0.7",
     "postcss-import": "^12.0.1",
-    "tailwindcss": "^1.9.6"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-hero-icon": "^0.0.7",
+    "react-router-dom": "^5.3.0",
+    "sanitize-html": "^2.5.0",
+    "tailwindcss": "^1.9.6",
```

Comment:
@rebeccahongsf any reason we couldn't upgrade to the latest of tw?

Reply:
No reason at all — let's use the latest version of TW! 😄  @nplowman would you be comfortable with updating TW within this PR?

Reply:
Yeah, updating should be fine as long as it doesn't introduce unintended regressions. I'll give it a try and see if there are any obvious issues.